### PR TITLE
Update v1.2.0 User Guide

### LINKED ISSUE #57: Update User Guide
As a new user, I can view the user guide easily so that I can learn more about the product as and when I need

### PR DESCRIPTION
**Pull request by erisjacey**
Features:
- Add
- Delete
- Find
- List
- Clear
- Exit

Coming soon:
- Help
- User confirmation prompt

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,176 +3,203 @@ layout: page
...
-   ![Ui](images/Ui.png)
+4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note how the app contains some sample data.<br>
+   [insert image of GUI here]
 
-1. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
+5. Type the command in the command box and press Enter to execute it. e.g. typing **`help`** and pressing Enter will open the help window.<br>
    Some example commands you can try:
 
-   * **`list`** : Lists all contacts.
+   * **`list`** : Lists out all the contacts.
```
Comment (nweiyue) [inline docs/UserGuide.md]: Perhaps "contacts" can be changed to "students"?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -3,176 +3,203 @@ layout: page
...
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`add n/John Cena e/thechamp@example.com`** : Adds a student named `John Cena` to the class list.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`delete 3`** : Deletes the 3rd contact shown in the current list.
```
Comment (nweiyue) [inline docs/UserGuide.md]: Perhaps "contact" can be changed to "student"?

File: docs/UserGuide.md [hunk 3]
```diff
@@ -3,176 +3,203 @@ layout: page
...
 
-   * **`clear`** : Deletes all contacts.
+   * **`clear`** : Deletes all the contacts.
```
Comment (nweiyue) [inline docs/UserGuide.md]: Perhaps "contacts" can be changed to "students"?

File: docs/UserGuide.md [hunk 4]
```diff
@@ -3,176 +3,203 @@ layout: page
...
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+```
+delete INDEX
+```
+
+* Deletes the person at the specified `INDEX`.
```
Comment (nweiyue) [inline docs/UserGuide.md]: Perhaps "person" can be changed to "student"? I noticed the same issue in several other places too.

**Review by nweiyue — COMMENTED** [review]
LGTM. Some nits to fix.

**Review by CodingCookieRookie — APPROVED** [review]
LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (erisjacey) [reply docs/UserGuide.md]: Thanks for pointing this out, as well as in other places!

**Pull request merged**